### PR TITLE
Allow "inner_hits" key in ES search body whitelist

### LINKED ISSUE #1956: Allow "inner_hits" key in Elasticsearch search body whitelist
Add the [inner_hits](https://www.elastic.co/guide/en/elasticsearch/reference/7.10/inner-hits.html) ES key for nested query inside our [whitelist](https://github.com/kuzzleio/kuzzle/blob/master/lib/service/storage/elasticsearch.js#L105)

### PR DESCRIPTION
**Pull request by MathieuVeber**
Closes #1956

## What does this PR do ?
Adding compatibility with inner_hits [see here](https://discord.com/channels/306054601436561410/694124702053957720/806623592246804531)

### CODE REVIEW

File: lib/config/default.config.js [hunk 1]
```diff
@@ -250,7 +250,7 @@ module.exports = {
       }
     },
     internalIndex: {
-      bootstrapLockTimeout: 5000
+      bootstrapLockTimeout: 60000
```
Comment (Aschen) [inline lib/config/default.config.js:253]: Why this change?

**Review by Aschen — CHANGES_REQUESTED** [review]
Documentation must be updated accordingly https://github.com/kuzzleio/kuzzle/blob/master/doc/2/guides/main-concepts/querying/index.md

File: lib/config/default.config.js (hunk 1 above)
Comment (MathieuVeber) [reply lib/config/default.config.js:253]: I don't know this is coming from @scottinet's commit

**Review by Aschen — APPROVED** [review]

File: lib/config/default.config.js (hunk 1 above)
Comment (scottinet) [reply lib/config/default.config.js:253]: Yes it does.
I cherry-picked a commit from the 2-dev branch to master to fix our CI a few days ago. I don't get why it appears here. :thinking: 
But it doesn't really matter.

**Review by scottinet — APPROVED** [review]
